sdl2promela/sdl/model.py

```diff
@@ -216,16 +221,16 @@ def __gather_inputs(self):
             input.transitions = {}
             self.inputs[input.name] = input
         # Build transition list
-        for state_name in self.source.mapping:
+        for state_name, input_list in self.source.mapping.items():
             target = self.states[state_name]
-            input_list = self.source.mapping[state_name]
             if not isinstance(input_list, List):
                 continue
             # TODO - adjust for multiple inputs in inputString
             for input in input_list:
                 id = input.transition_id
-                trigger = self.inputs[input.inputString]
-                trigger.transitions[id] = target
```

Comment:
Has input changed its meaning? basically, singleInput is an iterator of inputlist.inputlist.source.mapping.items. I think it would be good if the variable naming provided better hints.

Reply:
To be honest, I have no idea. I vhanged this code while I was debugging example with parameters using pdb, This works, but at this moment I don't remember why.

Reply:
input is an instance of opengeode.ogAST.Input`
From `ogAst.py`:
```
        ''' Initialize the Input attributes '''
        # inputString is the user text, it can contain several inputs
        self.inputString = ''
```
and:
```
        # list of input signal names
        self.inputlist = []
```
So, the inputString might be: `result(Pint)`, while the `inputlist` has value: `['result']`. In fact, the valid value for `inputString` is also `*` or `error,success`, while `inputlist` contains a parsed list of input signals.
I think this resolves `# TODO - adjust for multiple inputs in inputString`.

I will try to make this code more readable.

Reply:
Done